numpy/random/bit_generator.pyi

```diff
@@ -0,0 +1,61 @@
+from threading import Lock
+from typing import Literal, Optional
+
+from numpy import DtypeLike, ndarray
+from numpy.typing import ArrayLike, _ShapeLike
+
+class SeedlessSeedSequence:
```

Comment:
It seems that `SeedlessSeedSequence` (more or less) has `ISpawnableSeedSequence` as an abstract baseclass.
https://github.com/numpy/numpy/blob/5cae51e794d69dd553104099305e9f92db237c53/numpy/random/bit_generator.pyx#L250-L252

Reply:
I have typed these as ABC.  The typing versions of the class can use standard interitance as far as I can tell unlike the Cython version which need to register.